qiskit/opflow/converters/circuit_sampler.py

```diff
@@ -329,7 +329,7 @@ def sample_circuits(
             ready_circs = self._transpiled_circ_cache
 
         # run transpiler passes on bound circuits
-        if self._transpile_before_bind:
+        if self._transpile_before_bind and self.quantum_instance.bound_pass_manager:
```

Comment:
Using `None` as False has led to some bugs difficult to resolve in the past 🙂 
```suggestion
        if self._transpile_before_bind and self.quantum_instance.bound_pass_manager is not None:
```